linode/manager
# refactor: [M3-7022] - Make `<ActionMenu />` use MUI instead of Reach UI

## Description 📝

- Reach UI duplicates MUI features so we might as well just use MUI. This will help us consolidate styles and packages.

## Major Changes 🔄
- Makes `ActionMenu` use MUI Menu instead of Reach UI 🎉
- Uninstalls all Reach UI packages except `@reach/tabs` because we still depend on it 😔

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-08-11 at 3 27 17 PM](https://github.com/linode/manager/assets/115251059/8084d971-7b89-47f1-b284-ff3578a79c2a) | ![Screenshot 2023-08-11 at 3 27 44 PM](https://github.com/linode/manager/assets/115251059/2b28a563-a6bd-4f33-857b-98664e439175) |

## How to test 🧪
- Verify `<ActionMenu />`'s still work
- Verify `<ActionMenu />` styles look alright and close to production
  - I tried my best to style it similarly to production, but if you have an suggestions to make it closer to production, push to my PR or leave a comment! 🎨
  - Please test both **light** and **dark** mode
- Verify `<ActionMenu />` is accessible
